commit 20cd74104cd4e50100f5bf87bac9e7d255af954f
Author: Bohuslav Kabrda <bkabrda@redhat.com>
Date:   Tue Feb 26 14:53:51 2013 +0100

    Update spec/regression/JRUBY-5122_nonblocking_io_spec.rb
    
    Raise socket_channel_might_block to also block on Fedora 19, OpenJDK 7.0.

diff --git a/spec/regression/JRUBY-5122_nonblocking_io_spec.rb b/spec/regression/JRUBY-5122_nonblocking_io_spec.rb
index 10bdf8c..dc0b269 100644
--- a/spec/regression/JRUBY-5122_nonblocking_io_spec.rb
+++ b/spec/regression/JRUBY-5122_nonblocking_io_spec.rb
@@ -214,7 +214,7 @@ describe "nonblocking IO blocking behavior: JRUBY-5122" do
   #   Oracle's build block with > 131072 (2**17)
   # On a Windows 7(64) box:
   #   Oracle's build does not block (use memory till OOMException)
-  SOCKET_CHANNEL_MIGHT_BLOCK = "a" * (65536 * 4)
+  SOCKET_CHANNEL_MIGHT_BLOCK = "a" * (219463 * 4)
 
   it "should not block for write" do
   100.times do # for acceleration; it failed w/o wait_for_accepted call
